src/com/facebook/buck/cli/TestCommand.java

```diff
@@ -259,6 +259,11 @@ public int getNumTestManagedThreads(BuckConfig buckConfig) {
   }
 
   private TestRunningOptions getTestRunningOptions(CommandRunnerParams params) {
+    // this.coverageReportFormats should never be empty, but doing this to avoid problems with
```

Comment:
if it should never be empty, please add a check and throw a human readable exception telling users that they should pass the parameter :) Previously it would just throw an enumset exception and now it's going to be completely silent :)

Reply:
CoverageReportFormatsHandler will throw the human readable option exception in case of it being empty, that's why it will never be empty by this point.

Reply:
Also maybe I'm not configuring output verbosity correctly, but if I throw any exception from this context, buck just silently exits, that's why I Option handler returning user facing is preferable

Reply:
awesome, that sounds good!